### PR TITLE
updated hookup graphic and made the tex file not wrong

### PR DESCRIPTION
**Pull request by david-deboer**
I've added the WR to the diagram, and corrected the dated information.

### CODE REVIEW

File: Docs/mc_definition.tex [hunk 1]
```diff
@@ -110,13 +110,7 @@ \section{Configuration Management}
...
-\begin{itemize}\setlength\itemsep{-.3em}
-	\item not\_connected
-	\item needs\_checking
-	\item known\_bad
-	\item passed\_checks
-\end{itemize}
+To capture antenna status, an {\em a priori} antenna status table has been defined to facilitate a work flow to incorporate new antennas into the array and capture and known recalcitrant antennas.  The table provides an enumerated status for each antenna station.
 
 
 \subsection{Operation Workflow}
```
Comment (bhazelton) [inline Docs/mc_definition.tex]: The text in this subsection is out of date.

File: Docs/mc_definition.tex [hunk 2]
```diff
@@ -110,13 +110,7 @@ \section{Configuration Management}
 
 Parts are hooked together via connections of their ports, as defined in the connection table.   There are two types of connections:  (1) "hookup" connections and (2) "physical" connections, discussed below.
```
Comment (bhazelton) [inline Docs/mc_definition.tex]: Double quotes don't work properly in latex -- you need to use the backtick and single quotes to get them to go the correct direction.

File: Docs/mc_definition.tex [hunk 3]
```diff
@@ -174,6 +169,7 @@ \subsection{Scripts}
 \subsubsection{Geographical Information:  {\tt geo.py fg\_action --arguments}}
 
 Has various plotting/printing options for station information.  {{\tt fg\_action} is the "foreground action", that is the locations that will get printed and shown on top if plotting.
```
Comment (bhazelton) [inline Docs/mc_definition.tex]: Double quotes don't work properly in latex -- you need to use the backtick and single quotes to get them to go the correct direction.

File: Docs/mc_definition.tex (hunk 1 above)
Comment (david-deboer) [reply Docs/mc_definition.tex]: Updated.

File: Docs/mc_definition.tex (hunk 2 above)
Comment (david-deboer) [reply Docs/mc_definition.tex]: Fixed quotes and also added Arduino to drawing.

File: Docs/mc_definition.tex [hunk 4]
```diff
@@ -108,20 +108,25 @@ \section{Configuration Management}
 
 All parts and connections are timed in that they have a start and stop time of operation.  If stop is {\tt None}, then it is active (it is given a date in the relatively far future).  There are currently two special parts (one at each end of the signal chain) that are ``geo\_located'' parts and have entries in the geo\_location table: {\bf station} and {\bf node}.
 
-Parts are hooked together via connections of their ports, as defined in the connection table.   There are two types of connections:  (1) "hookup" connections and (2) "physical" connections, discussed below.
+Parts are hooked together via connections of their ports, as defined in the connection table.   There are two types of connections:  (1) ``hookup'' connections and (2) ``physica'' connections, discussed below.
```
Comment (bhazelton) [inline Docs/mc_definition.tex]: physica -> physical

**Pull request merged**